packages/gatsby-plugin-page-creator/src/create-path.js

```diff
@@ -1,11 +1,9 @@
 // @flow
 import parsePath from "parse-filepath"
```

Comment:
Our of curiosity, is there any reason `parse-filepath` is being used in place of the native `path.parse`? It seems this code would not be bundled.

If it would be helpful, I could follow up with a PR to remove that dependency.

Reply:
It doesn't look like any of the additional `parse-filepath` utilities are being used, so it makes sense to remove this dependency in favor of the native `path.parse`.

Go ahead 👍

Reply:
Cool, if @wardpeet agrees I'll do that after this gets merged.

Reply:
seems right, parse-filepath is a polyfill and parse.path is supported in all versions we support so please remove 👍